### ISSUE TITLE
"Missing top-level class documentation comment" with Rails 5 default files.

### ISSUE THREAD

**Issue by johnunclesam** [bbatsov/rubocop#5160]
## Expected behavior

If I use in my `.rubocop.yml` file this:

```
Rails:
  Enabled: true
```

I don't want in file like `app/controllers/application_controller.rb*` this warning: `Missing top-level class documentation comment.`

## Actual behavior

Now it happens.

Am I wrong?

## RuboCop version

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x64-mingw32)
```

**Comment by Drenmi** [comment]
Enabling the Rails department only adds some additional cops to the inspection. It does not include any Rails specific configuration for the regular cops. We try not to hard code special cases for libraries into RuboCop core.

I personally use this configuration on Rails projects:

```yaml
Style/Documentation:
  Include:
    - app/models/**/*
    - lib/**/*
```

Since I only find it helpful to document domain objects.

Of course you can tweak this to enforce configuration for the files you want. Hope this helps!

**Comment by johnunclesam** [comment]
Do you use VSCode with this config?

**Comment by manavm1990** [comment]
https://rubygems.org/gems/rubocop-rails
